calico_containers/tests/st/test_add_ip.py

```diff
@@ -13,7 +13,7 @@ def test_add_ip(self):
         """
         host = DockerHost('host')
 
-        host.execute("docker run -e CALICO_IP=192.168.1.1 -tid --name=node1 busybox", use_powerstrip=True)
+        host.create_workload("node1", "192.168.1.1")
         host.execute("docker run -tid --name=node2 busybox")
```

Comment:
Feels like this style of creating a workload should also be accommodated in `create_workload`